src/server/serverSetup.js

```diff
@@ -41,15 +46,69 @@ fluid.defaults("sjrk.storyTelling.server", {
                     secrets: "@expand:sjrk.storyTelling.server.resolveJSONFile({that}.options.secureConfig.secretsConfigPath)"
                 },
                 port: "{that}.options.globalConfig.port",
+                session: {
+                    store: "@expand:sjrk.storyTelling.server.makeMemorySessionStore({that}.options.globalConfig.sessionCheckPeriod)",
+                    middlewareOptions: {
+                        secret: "{server}.options.secureConfig.secrets.session"
+                    }
+                },
+                authorCredentialConfig: "{that}.options.secureConfig.secrets.authorCredentialConfig",
                 distributeOptions: {
-                    record: "@expand:kettle.resolvers.env(COUCHDB_URL)",
-                    target: "{that sjrk.storyTelling.server.dataSource.couch.core}.options.host"
+                    couchDBURL: {
+                        record: "@expand:kettle.resolvers.env(COUCHDB_URL)",
+                        target: "{that sjrk.storyTelling.server.dataSource.couch.core}.options.host"
+                    },
+                    couchDBAuthorsURL: {
+                        record: "@expand:kettle.resolvers.env(COUCHDB_URL)",
+                        target: "{that expressUserUtils}.options.dataSourceConfig.host"
+                    },
+                    sessionOptions: {
+                        source: "{that}.options.session",
+                        target: "{that > kettle.middlewareHolder > session}.options"
+                    },
+                    authorCredentialConfig: {
+                        source: "{that}.options.account",
```

Comment:
The differentiation of the kettle server config and account credential config in various config files makes a lot of sense. Thanks, @jobara.

With the renaming at line 55, the source reference of `account` should also be renamed to `authorCredentialConfig`. It would be nice to have tests for the proper handling/distribution of secrets.json.

Reply:
Thanks for catching the leftover bit. I've addressed that and added some test for the distributions of the secrets.